modules/ROOT/pages/setup-edit.adoc

```diff
@@ -0,0 +1,55 @@
+= Set Up Partner Manager
+
+Before you can deploy message flows from Partner Manager, you must set up and configure Anypoint Partner Manager for use by the team members in your organization who configure, deploy, and monitor B2B Trading Partner profiles and message flows.
+
+== Prerequisites
+
+Before you set up Partner Manager, you must have:
+
+* An Aypoint Platform user account with the Anypoint Partner Manager entitlement.
+* The Partner Manager Administrator role, which assigns permission levels for all users who need access to B2B message flows in their environment (Production or  Sandbox).
+* The required Mule vCore capacity to handle your B2B transaction workload needs. 
+* Licenses for the connectors that are used in your B2B message flows based on the protocols and message formats you are using:
+* AS2 connector
+* X12 connector
+* Anypoint Studio 7.4 or later in order to create DataWeave maps for EDI data transformation.
+
+If you require assistance, contact your account representative.
+
+== Storage Requirements
+
+Storage for the B2B transmission payloads you send or receive, such as a database, Amazon S3, Azure Blob Storage, and so on. +
+For more information, refer to xref:setup-payload-storage-API.adoc[Set up the Payload Storage API for Partner Manager].
+
+== Deployment Requirements
+
+* Mule runtime engine (Mule) 4.2.2 or later (for on-premises only).
+* Keep the default deployment of message flows to CloudHub (for information on CloudHub settings, refer to xref:cloudhub-deploy-options.adoc[CloudHub Deployment Options], or deploy to your own hosted on-premise Mule runtime, by clicking *Stop deploying to CloudHub* and going to  xref:deploy-onpremise.adoc[Deploy On-Premises for Partner Manager]).
```

Comment:
This is an instruction, not parallel to the noun list item above it.
Make list items parallel.
Possibility:
```suggestion
* Deployment of message flows to CloudHub (default)
+
For information about CloudHub settings, refer to xref:cloudhub-deploy-options.adoc[CloudHub Deployment Options]. To deploy to your own hosted on-premise Mule instance, click *Stop deploying to CloudHub* and refer to  xref:deploy-onpremise.adoc[Deploy On-Premises for Partner Manager]).
```
HOWEVER...this is an odd list item, because the cross-refs include instructions to the user, and that is confusing.
I suggest reworking to actually list requirements and requirements only:
* v4.2.2 or later on-premises
* Deployment environment
** CloudHub
** On-premises

Maybe this whole topic should be divided into CloudHub vs. On-prem deployment reqs?